### ISSUE TITLE
Support for Croatia's sources

### ISSUE THREAD

**Issue by fmis13** [Glodenox/wme-om#116]
Hello!
Is it possible to add any of these layers to the map? Please see below. All are under Croatia's open license (no restrictions, without attribution).
Some may require a free API key available by creating an account - is that possible to implement?
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/d4ffb43e-6713-459a-80f5-b04d44be6879
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/e8a9f098-1f07-4152-bd08-dfa6c96cd57e
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/bf71962f-55c5-4ee3-a525-b9a6b1b8f7af
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/cd006437-7e7e-496a-8372-2562ceced793
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/17df05e6-f650-4154-89fd-010be09a3f8b
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/d4ffb43e-6713-459a-80f5-b04d44be6879
https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/7adfd100-8795-4d11-8677-ff18b3d4f2e1
All should be available with WMS.
Best regards!

**Comment by Glodenox** [comment]
I've checked the first data source and I'm under the impression that at least that one could be used, if the correct attribution text is added to the information page in the Waze app. You can contact your community manager for this.
I'm not sure about the Google Translate translation of the following page, so please check this to be sure: https://data.gov.hr/otvorena-dozvola

**Comment by fmis13** [comment]
Hi, I have reached out to the CM. Will contact you after the request is proccesed.

**Comment by fmis13** [comment]
Hi @Glodenox , the request was **approved** by Waze Legal and the attribution was added on [Map Data Attribution](https://support.google.com/waze/answer/12075833?hl=en-mode) . Can you please attempt to add the sources above?
Best regards!

**Comment by fmis13** [comment]
Hi, I have fiddled around with the script, and made some sources work (Digitalni ortfolio 2021, and the 2022 equivalent). I have also added the "Adrese" source but it seems to only show dots? That should show HNs.
Can you please check the sources at [here](https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/e8a9f098-1f07-4152-bd08-dfa6c96cd57e) and [here](https://geoportal.nipp.hr/geonetwork/srv/hrv/catalog.search#/metadata/7adfd100-8795-4d11-8677-ff18b3d4f2e1) ?
Here is my copy of the script. Please check what I have changed since I don't have any experience with JavaScript.
https://hastebin.com/share/ruriquzile.javascript
Best regards!

**Comment by Glodenox** [comment]
Sorry for the delay on handling this, but I've finally found some time to work on this and just released these. By disabling the CORS-related setting for maps that can't or don't need to manipulate the map tiles when they are received, I'm able to call the map tiles without having to pass it by a proxy (which was giving me quota issues anyway).

I've now added about 6 maps, but honestly the amount of data in them is quite limited. As you've noticed yourself, there are no actual numbers in the address map (not just visual, it can't be queried either). The buildings map doesn't have any information on the buildings, and the transport network map doesn't contain any street names.
